snap/local/hooks/cmd/install/install.go

```diff
@@ -204,6 +207,24 @@ func installKuiper() error {
 		return err
 	}
 
+	// copy files from $SNAP_DATA/etc to $SNAP_DATA/kuiper
+	copyScriptPath, err := exec.LookPath("install-setup-kuiper.sh")
+	if err != nil {
+		return err
+	}
+
+	cmdCopyKuiper := exec.Cmd{
+		Path:   copyScriptPath,
+		Args:   []string{copyScriptPath},
+		Stdout: os.Stdout,
+		Stderr: os.Stdout,
+	}
+
+	err = cmdCopyKuiper.Run()
+	if err != nil {
+		return err
+	}
+
```

Comment:
The edge-ekuiper snap sets `KuiperBaseKey: $SNAP_DATA` while this snap has it set to `KuiperBaseKey: $SNAP_DATA/kuiper`. As a result, an additional script is called here to copy them (why not move) to copy ./etc from former to latter directory.

Can't we just make the [install hook](https://github.com/canonical/edgex-ekuiper-snap/blob/main/snap/hooks/install) of edgex-ekuiper take an environment variable as input base path which is set to `$SNAP_DATA` by default and set to `$SNAP_DATA/kuiper` by passing [Cmd.Env](https://pkg.go.dev/os/exec#Cmd) when called here?